src/Sarif/Writers/LogFilePersistenceOptions.cs

```diff
@@ -6,22 +6,20 @@
 namespace Microsoft.CodeAnalysis.Sarif.Writers
 {
     [Flags]
-    public enum LoggingOptions
+    public enum LogFilePersistenceOptions
+
```

Comment:
remove empty line

Reply:
Done.

---
In reply to: [561429474](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r561429474) [](ancestors = 561429474)